spec/features/show_page_spec.rb

```diff
@@ -1,6 +1,23 @@
 require "rails_helper"
 
 RSpec.describe "customer show page" do
+
+  it "displays the customers orders paginated" do
+    customer = create(:customer)
+    orders = []
+    10.times do
```

Comment:
I don't think we need to do this 10 times. That could slow the tests down a bit. You could do 2 orders and set `Kaminari.per_page = 2`. Something like:

```ruby
first_order, second_order = Array.new(2) { create :order }

visit admin_customer_path customer

expect(page).to have_content(first_order.id)
expect(page).to have_no_content(second_order.id)
click_on 'Next >'
expect(page).to have_content(second_order.id)
expect(page).to have_no_content(first_order.id)
```